QUALITY_DECLARATION.md

```diff
@@ -205,5 +205,3 @@ Comparing this table with the [Quality Level Comparison Chart of REP2004](https:
 Missing to quality level 3:
```

Comment:
The table above also needs updating.

Alternatively, it could be removed so the declaration matches most of the other ones.

Reply:
Good point.  I removed some pieces of this so it looks more like the other packages.